mex/model/vocabularies/consent-type.json

```diff
@@ -0,0 +1,40 @@
+[
+    {
+        "definition": {
+            "de": "Einwilligung, die indirekt durch eine Handlung gegebenen wurde, die nicht alleinig mit einer einwilligenden Entscheidung assoziert werden kann.",
+            "en": "Consent that is implied indirectly through an action not associated solely with conveying a consenting decision."
+        },
+        "exactMatch": [
+            "https://w3id.org/dpv#ImpliedConsent"
+        ],
+        "identifier": "https://mex.rki.de/item/consent-type-1",
+        "inScheme": "https://mex.rki.de/item/consent-type",
+        "note": {
```

Comment:
what is a "note"? we don't have that anywhere else. do we really need "definition", "note", and "$comment"?

Reply:
the vocab items follow the SKOS standard and this is supposed to be a `skos:note` (not a JSON schema `definition` or `$comment`). I added the property to the `concept` entity, see https://github.com/robert-koch-institut/mex-model/pull/49/commits/aa658d018e4cf6be72b1606f60e8ab0512346c50